docs/reference/crd-config/function-crd.md

```diff
@@ -121,6 +122,23 @@ Function Mesh provides Pulsar cluster configurations in the Function, Source, an
       </ul>
     </td>
   </tr>
+  <tr>
+    <td><code>cleanupAuthConfig</code></td>
+    <td>The authentication configurations for removing subscriptions and intermediate topics. You can configure generic authentication or <a href="https://oauth.net/">OAuth2 authentication</a> through this field. <p><b>Generic authentication</b></p>
```

Comment:
add `if not provided, the authConfig will be used`

Reply:
Updated. PTAL.